src/Components/button/button.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import './button.css'
+function Button ({ text, onClick }) {
+   
+      return (
+        <button  type="button" onClick={onClick} className="bg-primary hover:bg-green-700 text-white font-bold py-3 px-7 rounded-full " >
```

Comment:
can add shadows to the button, also the hover color isn't really nice if you can add some a bit brighter green it would be better